tests/test_import_glue.py

```diff
@@ -0,0 +1,98 @@
+import boto3
+from typer.testing import CliRunner
+import logging
+import yaml
+from moto import mock_aws
+import os
+import pytest
+
+from datacontract.cli import app
+from datacontract.data_contract import DataContract
+
+logging.basicConfig(level=logging.INFO, force=True)
+
+db_name = "test_database"
+table_name = "test_table"
+
+
+@pytest.fixture(scope="function")
+def aws_credentials():
+    """Mocked AWS Credentials for moto."""
+    os.environ["AWS_ACCESS_KEY_ID"] = "testing"
```

Comment:
Try using `monkeypatch.setenv()`

Reply:
I updated the file with monkeypath. Let's hope it works